float32 big_x
float32 big_y
float32 big_degree
int32[] big_action
int32[] action_list
int32[] cup_color
int32 cup
bool ns
bool team
float32 time
int32 status
